website_slides/__openerp__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# #############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    Copyright (C) 2014-TODAY Odoo SA (<https://www.odoo.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Slides',
+    'version': '1.0',
```

Comment:
How can I test this module please ? Thanks.

2015-10-07 14:03 GMT+01:00 Pedro M. Baeza <notifications@github.com>:

> In website_slides/__openerp__.py
> <https://github.com/OCA/website/pull/52#discussion_r41386669>:
>
> > +#    published by the Free Software Foundation, either version 3 of the
> > +#    License, or (at your option) any later version.
> > +#
> > +#    This program is distributed in the hope that it will be useful,
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#    GNU Affero General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU Affero General Public License
> > +#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +##############################################################################
> > +
> > +{
> > +    'name': 'Slides',
> > +    'version': '1.0',
>
> Version 8.0.1.0.0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/website/pull/52/files#r41386669>.
>



-- 

Cordialement

Yassine TEIMI

Consultant & Odoo Project Manager
TEL : +212 6-27 33 30 78